packages/app/src/components/Dimensions/DimensionItem.js

```diff
@@ -57,19 +69,21 @@ export class DimensionItem extends Component {
                 onMouseOver={this.onMouseOver}
                 onMouseLeave={this.onMouseExit}
             >
-                <DimensionLabel {...this.props}>
-                    {Icon}
+                <DimensionLabel {...this.props} isDeactived={this.isDeactived}>
+                    <div style={styles.iconWrapper}>{Icon}</div>
```

Comment:
Ideally these should be `className` instead of `style` (better for performance and flexibility) but we need to update that a lot more places.  It would be good to avoid adding new `style` attribute usage.